node/service/src/lib.rs

```diff
@@ -1324,13 +1358,25 @@ where
 							maybe_current_para_head?.encode(),
 						));
 
-						let additional_key_values = Some(vec![(
-							moonbeam_core_primitives::well_known_relay_keys::TIMESTAMP_NOW.to_vec(),
-							sp_timestamp::Timestamp::current().encode(),
-						)]);
+						// Get the mocked timestamp
+						let mut timestamp = 0u64;
+						TIMESTAMP.with(|x| {
+							timestamp = x.clone().take() + RELAY_CHAIN_SLOT_DURATION_MILLIS;
```

Comment:
Same as above, also this very weird read is confusing and useless, as take() mutate so at a first glance we think that this line mutate the global variable